Clang 16 warning:

```
gcc -DHAVE_CONFIG_H -I.  -DPKGDATADIR=\"/usr/local/share/arp-scan\" -DPKGSYSCONFDIR=\"/usr/local/etc/arp-scan\"   -g -O2 -Wall -Wextra -Wformat-security -Wshadow -Wwrite-strings -fstack-protector -D_FORTIFY_SOURCE=2 -MT arp-scan.o -MD -MP -MF .deps/arp-scan.Tpo -c -o arp-scan.o arp-scan.c
arp-scan.c:1748:13: warning: variable 'iterations' set but not used [-Wunused-but-set-variable]
 1748 |    unsigned iterations = 0; /* Used for debugging */
      |             ^
1 warning generated.
```

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,6 +1,7 @@
 2025-03-07 Roy Hills <royhills@hotmail.com>
 
 	* arp-scan.c: Removed unused variable _iterations_ from find_host(). Detected by Apple Clang version 16.0.0 (clang-1600.0.26.6).
+	  https://github.com/royhills/arp-scan/pull/200
 
 	* NEWS: Updated to 2025-02-12.
 

--- a/NEWS.md
+++ b/NEWS.md
@@ -28,6 +28,7 @@
   - Source tree tidy-up: move test data files into separate _testdata_ directory; moved local autoconf macros to seperate files under _m4_ (requires autoconf >= 2.70 to build - v1.10.0 required autoconf >= 2.69).
   - Require a compiler with C99 support.
   - Change the HTTP user agent string used by the _get_oui_ script to mimic Chrome on Windows 10/x64 because the IEEE site rejects requests with the default libwww-perl user agent.
+  - Removed "Ununsed variable" reported by Clang 16.0.
   - Various minor improvements to the code and documentation.
 
 # 2022-12-10 arp-scan 1.10.0 (git tag 1.10.0)

